pkg/agent/containerd/watcher.go

```diff
@@ -259,8 +259,7 @@ func importAndWatchImages(ctx context.Context, cfg *config.Node) error {
 
 	// wait for the workqueue to empty before returning
 	for w.workqueue.Len() > 0 {
-		logrus.Debugf("Waiting for initial import of images from %s", cfg.Images)
-		time.Sleep(time.Second * 2)
+		time.Sleep(500 * time.Millisecond)
```

Comment:
This change does not seem related to the intent of the commit, is this change perhaps an oversight?

Reply:
This is marginally related, in that it unconditionally blocks containerd startup for at least this long, and it was making my tests slower unnecessarily.